samples/word/accept_all_revisions/cs/Program.cs

```diff
@@ -19,62 +19,87 @@ static void AcceptAllRevisions(string fileName, string authorName)
+    {
+        element.Remove();
+    }
+}
 
-        insertions.AddRange(body.Descendants<InsertedMathControl>()
-            .Where(c => c.Author is not null && c.Author.Value == authorName).Cast<OpenXmlElement>().ToList());
+// Method to handle insertions in the document body
+static void HandleInsertions(Body body, string authorName)
+{
+    // Collect all insertion elements by the specified author
+    var insertions = body.Descendants<Inserted>().Cast<OpenXmlElement>().ToList();
+    insertions.AddRange(body.Descendants<InsertedRun>().Where(c => c.Author?.Value == authorName));
+    insertions.AddRange(body.Descendants<InsertedMathControl>().Where(c => c.Author?.Value == authorName));
 
-        foreach (OpenXmlElement insertion in insertions)
+    foreach (var insertion in insertions)
+    {
+        // Promote new content to the same level as the node and then delete the node
+        foreach (var run in insertion.Elements<Run>())
         {
-            // Found new content.
-            // Promote them to the same level as node, and then delete the node.
-            foreach (var run in insertion.Elements<Run>())
+
+            if (run == insertion.FirstChild)
+            {
+                insertion.InsertAfterSelf(new Run(run.OuterXml));
+            }
+            else
             {
-                if (run == insertion.FirstChild)
-                {
-                    insertion.InsertAfterSelf(new Run(run.OuterXml));
-                }
-                else
-                {
-                    OpenXmlElement nextSibling = insertion.NextSibling()!;
-                    nextSibling.InsertAfterSelf(new Run(run.OuterXml));
-                }
+                OpenXmlElement nextSibling = insertion.NextSibling()!;
+                nextSibling.InsertAfterSelf(new Run(run.OuterXml));
             }
+        }
 
-            insertion.RemoveAttribute("rsidR", "https://schemas.openxmlformats.org/wordprocessingml/2006/main");
-            insertion.RemoveAttribute("rsidRPr", "https://schemas.openxmlformats.org/wordprocessingml/2006/main");
-            insertion.Remove();
+        // Remove specific attributes and the insertion element itself
+        insertion.RemoveAttribute("rsidR", "https://schemas.openxmlformats.org/wordprocessingml/2006/main");
+        insertion.RemoveAttribute("rsidRPr", "https://schemas.openxmlformats.org/wordprocessingml/2006/main");
+        insertion.Remove();
+    }
+}
+
+// Method to handle move-to elements in the document body
+static void HandleMoveToElements(Body body, string authorName)
+{
+    // Collect all move-to elements by the specified author
+    var moveToElements = body.Descendants<MoveToRun>().Cast<OpenXmlElement>().ToList();
+    moveToElements.AddRange(body.Descendants<Paragraph>()
+        .Where(p => p.Descendants<MoveFrom>()
+        .Any(m => m.Author?.Value == authorName)));
+    moveToElements.AddRange(body.Descendants<MoveToRangeEnd>());
```

Comment:
```suggestion
    var paragraphs = body.Descendants<Paragraph>()
        .Where(p => p.Descendants<MoveFrom>()
          .Any(m => m.Author?.Value == authorName));
    var moveToRun= body.Descendants<MoveToRun>();
    var moveToRangeEnd = body.Descendants<MoveToRangeEnd>();
    
    var moveToElements = [.. paragraphs, .. moveToRun, .. moveToRangeEnd];
```

Reply:
AddRange feels out of place with all the LINQ stuff - better to stay in one camp or the other

Reply:
For the VB, could do the equivalent of

```
var moveToElements = paragraphs.Concat(moveToRun).Concat(moveToRangeEnd);
```